mypy/nodes.py

```diff
@@ -2674,7 +2681,7 @@ class is generic then it will be a type constructor of higher kind.
     is_abstract: bool  # Does the class have any abstract attributes?
     is_protocol: bool  # Is this a protocol class?
     runtime_protocol: bool  # Does this protocol support isinstance checks?
-    abstract_attributes: List[str]
+    abstract_attributes: List[Tuple[str, int]]
```

Comment:
Add a comment above explaining what is the meaning of this attribute (i.e. what is `int` there).